sycl/plugins/unified_runtime/ur/adapters/cuda/device.cpp

```diff
@@ -873,6 +873,85 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
   case UR_DEVICE_INFO_MAX_COMPUTE_QUEUE_INDICES: {
     return ReturnValue(int32_t{1});
   }
+  case UR_DEVICE_INFO_BINDLESS_IMAGES_SUPPORT_EXP: {
+    // On CUDA bindless images are supported
```

Comment:
```suggestion
    // On CUDA bindless images are supported.
```

Same elsewhere.

Reply:
Added punctuation to comments throughout.